doc/mana-centos-tutorial.txt

```diff
@@ -191,18 +200,84 @@ NOTE: In some cases, where there's an Nvidia GPU is installed, one may need to
   install extra libraries (e.g., libX11, libXau, libXext, libxcb) similarly
   as mentioned above.
 
+### CENTOS 8 STREAM ###
+
+These steps are largely similar to the ones above. The SCL repository no longer
+exists on CentOS 8 Stream, and instead the relevant tools are included in
+Development Tools.
+
+  # dnf install -y git
```

Comment:
Let's make # for heading and $ for commands.

Reply:
sure.